test/factories/scholarship_applications.rb

```diff
@@ -1,8 +1,7 @@
 FactoryGirl.define do
   factory :scholarship_application do
     reason 'MyText'
```

Comment:
This will also require an associated `Scholarship`, similar to how this needs an associated `User`.